modules/notDuplicate.js

```diff
@@ -0,0 +1,8 @@
+const container = document.getElementById('container');
+
+export default function notDuplicate() {
```

Comment:
- [ ] Kindly update this function declaration to use arrow function syntax instead of the `function` keyword.